src/geometry/quaternion.rs

```diff
@@ -757,6 +772,15 @@ impl<N: Real> Quaternion<N> {
     }
 }
 
+impl<N: Real> Inv for Quaternion<N> {
```

Comment:
Please remove this `impl` and use `.try_inverse` directly on the `left_div` and `right_div` methods above.
If we want to implement more num traits, this should be done on a separate PR that addresses all the nalgebra types instead of just one.